scripts/test_examples.sh

```diff
@@ -201,44 +201,457 @@ stopKafka
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '18', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '19', key: 'async-proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '5', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '6', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '7', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '8', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '10', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '11', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '12', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '13', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '14', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '15', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '16', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '17', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '18', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '19', key: 'proto-key', 
+Sending 0 sync Kafka notifications (protobuf) ...
+Sending 0 async Kafka notifications (protobuf) ...
+")
+
+#this is not working - I do not know why
+#cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf -messageCount=\"0\""
+#testOutput "${cmd}" "${expected}"
+
+#result is:
+#time="2017-09-29 14:23:04.54780" level=info msg="Agent AfterInit took 0s" durationInNs=0 loc="core/agent_core.go(124)" logger=core tag=00000000 
+#time="2017-09-29 14:23:04.54782" level=error msg="Error loading core: plugin KafkaExample: Init error ''messageCount' has to be a number, not "0"', took 44.642�s" loc="core/event_loop.go(28)" logger=core tag=00000000 
+#Testing examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf -messageCount="0"
+
+#workaround
+cmd='examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf --messageCount 0'
+testOutput "${cmd}" "${expected}"
+
+
+# Let us test - in example one new message generated
```

Comment:
It seems this test does the same as the one above, i think we can keep just one of them

Reply:
The results of both discussed are same, but the first one is called without parameter and generating new messages and the second one is called with parameter --messageCount 0 - only lists the messages which were generated to that point.

I suggest to shorten the list of asserted messages and leave this second case.